src/applications/personalization/profile/components/ProfileSubNav.jsx

```diff
@@ -65,7 +65,6 @@ const ProfileSubNav = ({ isInMVI, isLOA3, routes, clickHandler = null }) => {
         header="Profile"
         icon-background-color="vads-color-primary"
         icon-name="account_circle"
-        role="navigation"
```

Comment:
This was unneeded as there is a nav inside the shadow DOM of VaSidenav with an aria-label of `Related pages menu`. I've confirmed VoiceOver is able to see it in navigation landmarks.